pyaedt/edb_core/EDB_Data.py

```diff
@@ -1723,6 +1723,70 @@ def material(self, materialname):
         self.edb_padstack.SetData(newPadstackDefinitionData)
 
 
+class EDBPadstackInstance(object):
+    """Manages EDB functionalities for a padstack.
+
+    Parameters
+    ----------
+    edb_padstackinstance :
+
+    _pedb :
+        Inherited AEDT object.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edb = Edb(myedb, edbversion="2021.2")
+    >>> edb_padstack_instance = edb.core_padstack.padstack_instances[0]
+    """
+
+    @property
+    def padstack_definition(self):
+        return self._edb_padstackinstance.GetPadstackDef().GetName()
+
+    @property
+    def backdrill_top(self):
+        layer = self._pedb.edb.Cell.Layer("", 1)
```

Comment:
Please add docstrings to all properties and methods you added to describe content

Reply:
Done